t/discovery/consul_dump.t

```diff
@@ -161,6 +165,8 @@ discovery:
   consul:
     servers:
       - "http://127.0.0.1:38500"
+    skip_services:
+      - "consul"
```

Comment:
Do we need to add it as the default value?

Reply:
the `consul` will register itself

Reply:
So the `consul` will be always registered automatically? If this is true, I think it would be better to skip by default. Otherwise, the users need to do this themselves.

Reply:
Yes, consul will be registered by default. I dont think it would be better if we skip it by default. We can give users to choice whether to skip or not, that can be closed-loop control. Of course, we should mention this info in Consul doc

Reply:
Is it a good idea? If consul is not skipped by default, people will need to repeat the same configuration each time they need to use consul discovery, similar to the `proxy_http_version 1.1` in the Nginx conf.

Reply:
Does the following scenario exist?
User A want to set up Consul route in APISIX with the Consul discovery plugin

Reply:
If you insist that people won't need to handle this case, it's OK.
But we need to write it down in the doc at least.

Reply:
OK, I add the `consul` as default skip service, and write it down in the doc